presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/db/DbFunctionNamespaceManagerConfig.java

```diff
@@ -52,25 +39,6 @@ public DbFunctionNamespaceManagerConfig setFunctionNamespacesTableName(String fu
         return this;
     }
 
-    @NotNull
-    public List<CatalogSchemaName> getDefaultFunctionNamespaces()
```

Comment:
So we will not provide a convenient initialization from config files anymore? People have to manually insert values to the db?

Reply:
I don't think this configuration property is that necessary since admin can just insert values into their DBs. It is also not an exactly a good design since it would be better to create function namespaces (whether the initial ones or not) at system level so it goes through proper code paths and checks, instead of in each "connector's" configuration. What just described will be supported by the `CREATE SCHEMA` statement.

Reply:
Make sure this is documented then.